Content.Server/Disposal/Unit/EntitySystems/DisposableSystem.cs

```diff
@@ -136,9 +136,9 @@ public void ExitDisposals(EntityUid uid, DisposalHolderComponent? holder = null,
                 {
                     _xformSystem.AttachToGridOrMap(entity, xform);
 
-                    if (holder.PreviousDirection != Direction.Invalid && _xformQuery.TryGetComponent(xform.ParentUid, out var parentXform))
+                    if (holder.PreviousDirection != Direction.Invalid && _xformQuery.TryGetComponent(uid, out var parentXform))  // start DeltaV EE Disposal fix
```

Comment:
just leave it how it was

Reply:
the item shot just dissapears if i dont do this change or the one prev committed

Reply:
the pipe's WorldRotation and the parent (grid) Should be the exact same, are you sure they are different

Reply:
I'm not sure of anything but when I changed it to only edit on line  141, no edit on 139, the objects sit in one place then eventually disappear